OpenRA.Game/Settings.cs

```diff
@@ -142,6 +142,7 @@ public class SoundSettings
 
 		public bool CashTicks = true;
 		public bool Mute = false;
+		public bool DisplayAltTexts = false;
```

Comment:
IMO it is better false in this case for 2 reasons - this option is remembered also, so user can check it once and it doesn't effect all current users. More important, there is one issue with such Mute+DisplayAltTexts default atm - there is "ChatLine" sound for each message when not muted. It works best either: 1. Muted + AltTexts on or off 2. Not Muted + AltTexts off.
Let me know please what you prefer.